assets/js/github-issues.js

```diff
@@ -81,7 +81,7 @@ $(document).ready(function () {
         },
       },
     ],
-    initComplete: () => {
+    initComplete() {
```

Comment:
`initComplete` here is an object member, it shouldn't be a regular function
see here: https://datatables.net/reference/option/initComplete

Reply:
Wait, isn't the function notation in member just syntactic sugar for `initComplete: function() {}`?

Reply:
Yes, the result should be the same even if I would prefer the longer version in this case, but it is only a stylistic matter.
Try to run the following snippet in the console and you should see the same result.
```
const x = { a() { console.log(this); }}; x.a();
const y = { a: function () { console.log(this); }}; y.a();
```

Reply:
No, wait, with the arrow function the function scope is bound to the parent one, and, if I understood correctly, that causes the undefined `this`. Probably because `this` is undefined in the parent scope.

Reply:
> No, wait, with the arrow function the function scope is bound to the parent one, and, if I understood correctly, that causes the undefined `this`. Probably because `this` is undefined in the parent scope.

Arrow function will bound `this` to its father, isn't it on purpose? Maybe I misunderstood

Reply:
Parent's `this` is window or something like that. We need datatable object's `this`.

Reply:
got it, so something like
`initComplete: function() {...},`
should works